handbook/engineering/campaigns/goals.md

```diff
@@ -13,6 +13,32 @@ What will get us there?
 - **Better debugging experience:** It can be challenging to debug a more complex campaign. Figuring out what went wrong and how to fix it is a slow iterative loop.
 - **User validation:** The above issues were revealed by user testing. Once we believe we have fixed them, we need to validate this with further user testing.
 
+
+### Design and build the best solution for changing code across many repositories
```

Comment:
I think the wording of this goal needs some work. This is specifically about the experience of *creating* a campaign. Maybe something like, "Design and build the best experience for creating changesets across many repositories"?